snippets/zoe/contracts/test-loan.js

```diff
@@ -0,0 +1,141 @@
+// @ts-check
+
+import '@agoric/install-ses';
+import { makeFakeVatAdmin } from '@agoric/zoe/src/contractFacet/fakeVatAdmin';
+import { makeZoe } from '@agoric/zoe';
+import bundleSource from '@agoric/bundle-source';
+import { makeIssuerKit } from '@agoric/ertp';
+import test from 'ava';
+import { E } from '@agoric/eventual-send';
+import buildManualTimer from '@agoric/zoe/tools/manualTimer';
+import { makeFakePriceAuthority } from '@agoric/zoe/tools/fakePriceAuthority';
+import { makeSubscriptionKit } from '@agoric/notifier';
+
+test('loan contract', async t => {
+  const zoe = makeZoe(makeFakeVatAdmin().admin);
+
+  const contractBundle = await bundleSource(
+    require.resolve('@agoric/zoe/src/contracts/loan'),
+  );
+  const installation = await E(zoe).install(contractBundle);
+
+  const collateralKit = makeIssuerKit('moola');
+  const loanKit = makeIssuerKit('simoleans');
```

Comment:
I think we have many kinds of Kits. Can these be e.g. `loanIssuerKit`? Or would it make more sense to replace `loanKit.issuer` with `loanIssuer`, and `loanKit.amountMath` with `loanAmountMath`?

Reply:
sure thing